packages/editor/src/client/routes/memberPlanList.tsx

```diff
@@ -191,7 +191,7 @@ export function MemberPlanList() {
         <Modal.Body>
           <DescriptionList>
             <DescriptionListItem label={t('memberPlanList.name')}>
-              {currentMemberPlan?.name || t('untitled')}}
+              {currentMemberPlan?.name || t('untitled')}
```

Comment:
Maybe nest this as we have made with the other translations? E.g. `t('memberPlans.overview.untitled')`